README.md

```diff
@@ -55,19 +55,24 @@ It's possible to run Panoptes only having to install the `fig_rake` gem. Alterna
 0. `cd` into the cloned folder. Run either `bundle install` or `gem install fig_rake`
 
 0. Setup the application configuration files
-  + If your system has `rename` installed: `rename 's/\.yml.hudson$/\.yml/' config/*.yml.hudson`
```

Comment:
I think that's a matter of personal preference. To me a dirty working dir doesn't matter, since I'm not going to `git add .` 

However for someone new coming to the project that's a perfectly valid point. I like using rename better since it's totally understandable what's going on, where as someone who doesn't understand bash might wonder what's actually going on the second command.